src/test/java/redis/clients/jedis/mocked/unified/UnifiedJedisTestBase.java

```diff
@@ -0,0 +1,73 @@
+package redis.clients.jedis.mocked.unified;
+
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import org.junit.After;
+import org.junit.Before;
+import org.mockito.Mock;
+import redis.clients.jedis.CommandObject;
+import redis.clients.jedis.CommandObjects;
+import redis.clients.jedis.PipeliningBase;
+import redis.clients.jedis.UnifiedJedis;
+import redis.clients.jedis.executors.CommandExecutor;
+import redis.clients.jedis.graph.GraphCommandObjects;
+import redis.clients.jedis.mocked.MockCommandObjectsTest;
+import redis.clients.jedis.providers.ConnectionProvider;
+
+/**
+ * Base class for {@link UnifiedJedis} mocked unit tests. Exposes a {@link UnifiedJedis} instance that
+ * uses mocked executors, providers and command objects, which can be asserted upon.
+ */
+public abstract class UnifiedJedisTestBase extends MockCommandObjectsTest {
+
+  /**
+   * The {@link UnifiedJedis} instance under-test.
+   */
+  protected UnifiedJedis unifiedJedis;
```

Comment:
```suggestion
  protected UnifiedJedis jedis;
```
This variable is likely being used several thousand times. A shorter name is easier to use and, more importantly, read.

Reply:
Renamed.